sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/DcosConstants.java

```diff
@@ -17,14 +17,14 @@
     public static final Boolean DEFAULT_GPU_POLICY = true;
     public static final String DEFAULT_IP_PROTOCOL = "tcp";
     public static final String DEFAULT_OVERLAY_NETWORK = "dcos";
+    public static final String DEFAULT_BRIDGE_NETWORK = "mesos-bridge";
     @edu.umd.cs.findbugs.annotations.SuppressFBWarnings
-    public static final Set<String> SUPPORTED_OVERLAY_NETWORKS = new HashSet<>(Arrays.asList(DEFAULT_OVERLAY_NETWORK));
+    public static final Set<String> SUPPORTED_OVERLAY_NETWORKS = new HashSet<>(
```

Comment:
1: No, because we don't have test clusters with anything other than `dcos` and `mesos-bridge`.
2: Yes there is a check and warning message.